src/modules/battery_status/analog_battery.cpp

```diff
@@ -1,3 +1,37 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2019 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 *   Copyright (c) 2020 PX4 Development Team. All rights reserved.
```

Reply:
It's from 2019, he just forgot to add the notice 😛

Reply:
I made it 2019-2020 since there are new changes as well.